src/Uno.UI/UI/Xaml/Markup/Reader/XamlObjectBuilder.cs

```diff
@@ -453,7 +454,7 @@ private void ProcessMemberMarkupExtension(object instance, XamlMemberDefinition
 			{
 				ProcessBindingMarkupNode(instance, member);
 			}
-			else if (IsStaticResourceMarkupNode(member))
+			else if (IsStaticResourceMarkupNode(member) || IsThemeResourceMarkupNode(member))
 			{
 				ProcessStaticResourceMarkupNode(instance, member);
```

Comment:
Should this be renamed to `ProcessStaticOrThemeResourceMarkupNode` to better indicate what it does?